apps/hash-frontend/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page/definition-tab.tsx

```diff
@@ -36,7 +36,10 @@ export const DefinitionTab = ({
       ...possiblyIncompletePropertyTypeOptions,
       ...entityTypeAndPropertyTypes.propertyTypes,
     };
-  }, [entityTypeAndPropertyTypes, possiblyIncompletePropertyTypeOptions]);
+  }, [
+    entityTypeAndPropertyTypes.propertyTypes,
+    possiblyIncompletePropertyTypeOptions,
```

Comment:
Nothing wrong with this change but not immediately clear what motivated it – could you explain?

Reply:
I made this change whilst trying to figure out what might be preventing the property types from being displayed in the entity type editor when loaded from a draft entity. I decided to keep it, as this effect technically only depends on the reference to the `entityTypeAndPropertyTypes.propertyTypes` changing.

Previously it was listening for all changes to the reference to `entityTypeAndPropertyTypes`. This could also prevent a bug if there's ever a situation where the reference to `entityTypeAndPropertyTypes` stays the same, but `entityTypeAndPropertyTypes.propertyTypes`. Not sure how likely that would've been though.